schema/classes/Asset.yaml

```diff
@@ -0,0 +1,61 @@
+"@context":
+  schema: http://schema.org/
+  rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  owl: http://www.w3.org/2002/07/owl#
+  void: http://rdfs.org/ns/void#
+  dcterms: http://purl.org/dc/terms/
+  dcat: http://www.w3.org/ns/dcat#
+  dcmitype: http://purl.org/dc/dcmitype/
+  dandi: http://schema.dandiarchive.org/
+  prov: http://www.w3.org/ns/prov#
+  crypto: http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/
+  md5: crypto:md5
+  sha1: crypto:sha1
+  sha256: crypto:sha256
+  sha512: crypto:sha512
+  attributedTo: prov:wasAttributedTo
+"@graph":
+  - "@id": dandi:Asset
+    "@type": rdfs:Class
+    rdfs:comment: An item describing the contents of a binary blob or file.
+    rdfs:label: Asset
+  - "@id": schema:name
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:contentUrl
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:contentSize
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:encodingFormat
+    schema:domainIncludes: dandi:Asset
+  - "@id": md5
+    schema:domainIncludes: dandi:Asset
+  - "@id": sha1
+    schema:domainIncludes: dandi:Asset
+  - "@id": sha256
+    schema:domainIncludes: dandi:Asset
+  - "@id": sha512
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:schemaVersion
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:identifier
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:sameAs
+    schema:domainIncludes: schema:Thing
+  - "@id": dandi:access
+    schema:domainIncludes: dandi:Asset
+  - "@id": dandi:relatedResource
+    schema:domainIncludes: dandi:Asset
+  - "@id": schema:version
```

Comment:
what does it mean for an asset to have a version?

Reply:
if you look at this, one may update the values of some of these properties. i could change the filename, add a relatedResource, etc.,. this would generate a new version of the asset. if we use a unique identifier when we generate an asset, then version is not relevant since the identifier uniquely carries version. however, if we don't then version becomes relevant. in schema.org, `version` is a property of `CreativeWork` and hence copied over here as a suitable thing for an asset/dandiset. we can consider this to be optional depending on how we represent the identifier of the asset.

Reply:
removed it.